src/components/input-time-picker/input-time-picker.stories.ts

```diff
@@ -37,4 +37,21 @@ export const DarkTheme = (): string => html`
   </calcite-input-time-picker>
 `;
 
+export const KoreanLocale = stepStory(
```

Comment:
Nice! Can you also add a story for Arabic or Hebrew + RTL?

Reply:
the direction shouldn't be specified as we don't support page direction right?